packages/react/menu/src/Menu.tsx

```diff
@@ -186,11 +188,9 @@ const MenuSubTrigger = React.forwardRef((props, forwardedRef) => {
         onMouseMove={composeEventHandlers(props.onMouseMove, () => {
           if (!props.disabled) context.onOpenChange(true);
         })}
-        onKeyDown={composeEventHandlers(props.onKeyDown, (event) => {
-          if (!props.disabled && SUB_OPEN_KEYS[context.dir].includes(event.key)) {
-            context.onKeyOpen();
-          }
-        })}
+        onSearchExclusiveKeyDown={(event) => {
```

Comment:
Looking at this `onSearchExclusiveKeyDown` implementation now, I'm starting to think we probably don't need it. It seems we can use `onKeyDownCapture` on `content` to `stopPropagation` when we're searching instead... something like this: https://codesandbox.io/s/heuristic-albattani-p1u0v?file=/src/App.js

Reply:
Okay ignore this, you're right, we did need this because we needed the custom event so we don't prevent consumer events... keep forgetting that part 🤯